### DISCUSSION TITLE
How to use a vim colorscheme

### DISCUSSION THREAD

**Discussion by fibbo** [LazyVim/LazyVim#3955] [Q&A]
I want to use the `alucard` version of Dracula Pro 2.0. I follow the instructions in the `INSTALL.md` but I don't get the theme working:

The steps I'm following summarized for neovim:
1. copy the theme into `~/.local/share/nvim/site/pack/themes/start/dracula_pro`
2. edit `init.vim` and add following lines:
```
" For packages, versions 8.2 and later will autoload `start` packages
" correctly even in your vimrc.
if v:version < 802
    packadd! dracula
endif
syntax enable
let g:dracula_colorterm = 0
colorscheme dracula_pro
```

As I have read that I shouldn't have `init.vim` and `init.lua` I pack this into a `vim.cmd([[]])` to call.
```
-- return early if we are within vs code
if vim.g.vscode ~= nil then
  return
end
vim.cmd([[
  syntax enable
  let g:dracula_colorterm = 0
  colorscheme alucard
]])
-- bootstrap lazy.nvim, LazyVim and your plugins_str
require("config.lazy")
```

But this just doesn't work. There is no error but also the theme doesn't change.

**Comment by arthureggert** [comment]
Same @fibbo did you make this work?

**Comment by fibbo** [comment]
I didn't, but I also didn't have any time to invest in finding a solution.

**Comment by bstncartwright** [comment]
I was able to get this working today! It requires you to have access to the `dracula-pro` GitHub organization ([you can request here](https://draculatheme.com/pro/request-access)) and your local git configured to use authentication that can access it. Then, use the following plugin configuration:
```lua
return {
  { "dracula-pro/vim" },

  {
    "LazyVim/LazyVim",
    opts = {
      colorscheme = function()
        vim.cmd("colorscheme dracula_pro_alucard")
      end,
    },
  },
}
```

Working great for me, and will keep the theme updated too!

**Reply by arthureggert** [reply]
@bstncartwright THIS IS AMAZING, thanks a lot!

**Reply by lucasvienna** [reply]
This worked great for me as well! Ended up with the following config:
```lua
return {
	{ "git@github.com:dracula-pro/vim.git", name = "dracula_pro" },
	{
		"LazyVim/LazyVim",
		opts = {
			colorscheme = "dracula_pro",
		},
	},
}
```
